docs/dotnet/dotnet-aspnet-metrics.md

```diff
@@ -0,0 +1,196 @@
+  * [Metric: `aspnet.rate_limiting.queued_requests`](#metric-aspnetrate_limitingqueued_requests)
+  * [Metric: `aspnet.rate_limiting.request.time_in_queue`](#metric-aspnetrate_limitingrequesttime_in_queue)
+  * [Metric: `aspnet.rate_limiting.requests`](#metric-aspnetrate_limitingrequests)
+
+<!-- tocstop -->
+
+## Server
+
+## Routing
+
+All routing metrics are reported by `Microsoft.AspNetCore.Routing` meter.
+
+### Metric: `aspnet.routing.match_attempts`
+
+<!-- semconv metric.aspnet.routing.match_attempts(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `aspnet.routing.match_attempts` | Counter | `{match}` | Number of requests that were attempted to be matched to an endpoint. [1] |
+
+**[1]:** Meter name is `Microsoft.AspNetCore.Routing`.
+<!-- endsemconv -->
+
+<!-- semconv metric.aspnet.routing.match_attempts(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `aspnet.routing.match_status` | string | Match result - success or failure | `success`; `failure` | Recommended |
+| `aspnet.routing.fallback` | string | TODO | `TODO` | Required |
+| `http.route` | string | The matched route (path template in the format used by the respective server framework). See note below [1] | `/users/:userID?`; `{controller}/{action}/{id?}` | Required |
+
+**[1]:** MUST NOT be populated when this is not supported by the HTTP server framework as the route attribute should have low-cardinality and the URI path can NOT substitute it.
+SHOULD include the [application root](/docs/http/http-spans.md#http-server-definitions) if there is one.
+
+`aspnet.routing.match_status` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `success` | Match succeeded |
+| `failure` | Match failed |
+<!-- endsemconv -->
+
+## Exception metrics
+
+Metrics reported by `Microsoft.AspNetCore.Diagnostics` meter.
+
+### Metric: `aspnet.diagnostics_handler.exceptions`
+
+<!-- semconv metric.aspnet.diagnostics_handler.exceptions(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `aspnet.diagnostics_handler.exceptions` | Counter | `{exception}` | Number of exceptions caught by exception handling middleware. [1] |
+
+**[1]:** Meter name is `Microsoft.AspNetCore.Diagnostics`
+<!-- endsemconv -->
+
+<!-- semconv metric.aspnet.diagnostics_handler.exceptions(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `aspnet.exception.result` | string | ASP.NET Core exception middleware handling result | `handled`; `unhandled` | Required |
+| `aspnet.handler` | string | TODO | `TODO` | Required |
```

Comment:
Full type name of the IExceptionHandler implementation that handled the exception.

Reply:
Also it's conditional on whether the exception was handled by a handler.

Reply:
would it make sense to drop `aspnet.exception.result`? then lack of a handler means it was unhandled

Reply:
> would it make sense to drop aspnet.exception.result? then lack of a handler means it was unhandled

Unhandled is one of [four possible values](https://github.com/lmolkova/semantic-conventions/pull/1/files#diff-a5d6f26f3c4f47136a0271839f8218277771fe17724b3ea05a590238d3a506e7R82). The presence or absence of handler won't distinguish all 4 cases.